### ISSUE TITLE
Accessibility feedback - Usability Testing Prep - Healthcare [Questionnaires MVP Priorities]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12176]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Jennifer (VSA) & Trevor (VSP)

## Description

This ticket houses accessibility issues for the usability testing prep review of Healthcare - Questionnaires MVP Priorities.

### Consider

* Noted: "This study requires participants to drag and drop elements and requires screensharing." Since this is a card sort and there is a possibility of having participants who have mobility or vision issues join, **consider** preparing an alternate approach or discovery if a participant is unable to use the card sort interface. As discussed in the review, having the backup plan you stated sounds great, and a listening session could also be valuable to understand the experience and needs of people with disabilities. For example: 
    - Do you remember the last time you visited VA.gov? 
    - What were you trying to do? 
    - What happened? 
    - What went through your mind when that happened? 
    - What were you expecting?
    - Is there a website that does a great job of serving your needs? If yes, what about it?

## Definition of done

1. Review and acknowledge feedback.
1. Document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions.

cc @1Copenut ready for review

**Comment by 1Copenut** [comment]
@jenstrickland VSP has no additional feedback. We are 💯

**Comment by abuckley3385** [comment]
Removing myself as assigned as these are Usability testing questions and should be assigned to @arickalewis1  for review and comment.

**Comment by arickalewis1** [comment]
Thanks for the feedback. I've reviewed it and we will proceed with listening sessions if Veterans are not able to use the card sort tool.
